services/postgresql/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 > **The STACKIT PostgreSQL service will reach its end of support on June 30th 2024. All calls done to the API after that will stop working.**
 >
 > Use `github.com/stackitcloud/stackit-sdk-go/services/postgresflex` instead. For more details, check https://docs.stackit.cloud/stackit/en/bring-your-data-to-stackit-postgresql-flex-138347648.html.
+## v0.14.0 (2024-05-13)
+
+- **Feature**: New method `GetMetrics` to get the latest metrics for cpu load, memory and disk usage for an instance
+- **Feature**: New method `ListBackups` to list the backups for an instance
```

Comment:
These methods were not added to `postgresql` right?

Reply:
Do we want to make more `postgresql` releases?

Reply:
Ups, overlooked it. 

> Do we want to make more postgresql releases?
I was wondering the same, since there were some changes I thought about doing it to keep support until the deprecation, but actually there is no new functionality so let's no make a release. I'll just merge the PR